include/bitcoin/explorer/commands/qrcode.hpp

```diff
@@ -125,7 +127,8 @@ class BCX_API qrcode
     virtual arguments_metadata& load_arguments()
     {
         return get_argument_metadata()
-            .add("PAYMENT_ADDRESS", 1);
+            .add("PAYMENT_ADDRESS", 1)
+            .add("filename", 1);
```

Comment:
I guess I knew that, since I already looked at the code :?. That's good, but still we prefer to pipe output vs. writing it to files.